### ISSUE TITLE
Tie Guild Crests to steam IDs

### ISSUE THREAD

**Issue by greatguys1** [MSRevive/MSCScripts#24]
Currently crests are obtained via a password. If the password is leaked, those who are not part of a guild can get guild crests.
Change to use no passwords. Use steam id to verify if a player should receive a crest after talking to the guild master.

Should guild leads only be able to spawn crests?
Should guild leads be able to promote members, who can also spawn crests?

**Comment by fegged** [comment]
Guild leads should be the sole person to spawn their crest, promoting may be beneficial.

Is Guild Master (the map) even necessary if you could theoretically spawn one immediately? Tho that code might get messy.

**Comment by greatguys1** [comment]
It isn't necessary, but the guild hall could be refactored into something that could be used for those that are in a guild. Although what it could be yet I'm not sure.

**Comment by BerntA** [comment]
Could this be something we could integrate in FN? Anyone part of the guild (registered in FN by steamID) can get the crest, and invite plrs to the guild so that they also may get the crest, but if the plr leaves the guild, the crest will be destroyed on spawn?

**Comment by greatguys1** [comment]
If practical, I think guild members should be registered via FN. Collecting crests is pretty fun, it might be cool to let other non-guild members keep any crests they have or are given.

**Comment by SaintWish** [comment]
> If practical, I think guild members should be registered via FN. Collecting crests is pretty fun, it might be cool to let other non-guild members keep any crests they have or are given.

Yeah people even when not part of the guild should be able to keep the crests because there's people that collect them. It could be cool to rewrite the guild system to be a proper one with officers and such, but not sure if that is something that is best handles by the FN if we do that then people can't play in single player and such?
